src/routes/github/create-branch/github-create-branch-options-get.ts

```diff
@@ -21,8 +21,7 @@ export const GithubCreateBranchOptionsGet = async (req: Request, res: Response,
 		return next(new Error(Errors.MISSING_ISSUE_KEY));
 	}
 
-	const jiraHost = getJiraHostFromTenantUrl(tenantUrl) || jiraHostQuery;
-
+	const jiraHost = prependProtocolToSiteName(res.locals.jiraHost) || prependProtocolToSiteName(tenantUrl) || jiraHostQuery;
 	// TODO move to middleware or shared for create-branch-get
```

Comment:
I think , prepend protocol should not be done here. If required, it should be done when reading `jirahost` from token.